### PR TITLE
making tablesorter size parser more robust

### LINKED ISSUE #1221: the tablesorter fails when the size of a file contains ","
Sometimes the "," is used as a delimiter between thousands. So for every file where the `size is >= 1000 and <= 1023` regardless the concrete units the tablesorter fails taking with him also all the other javascript beauty we have.

### PR DESCRIPTION
**Pull request by tulinkry**
fixes #1221

### CODE REVIEW

File: web/js/tablesorter.parsers.js [hunk 1]
```diff
@@ -45,8 +45,17 @@ $.tablesorter.addParser({
         return false;
     },
     format: function (s) {
-        var parts = s.match(/^([\d\.]+) ?(\w*)$/);
-        var num = parts[1];
+        /*
+         * This correctly handles thousand separator
+         * in a big number (either ',' or ' ' or none)
```
Comment (vladak) [inline web/js/tablesorter.parsers.js]: I'd maybe add bunch of concrete examples.

File: web/js/tablesorter.parsers.js [hunk 2]
```diff
@@ -45,8 +45,17 @@ $.tablesorter.addParser({
...
+         */
+        var parts = s.match(/^((?:\d{1,3}(?:[, ]?\d{1,3})*(?:\.\d+)?|\.\d+)) ?(\w*)$/);
```
Comment (vladak) [inline web/js/tablesorter.parsers.js]: Seems like there is extra paren level ?

```
((?:\d{1,3}(?:[, ]?\d{1,3})*(?:\.\d+)?|\.\d+))
^^                                          ^^
```

**Comment by vladak** [comment]
Pls also create new issue to get this tested (#1233).

File: web/js/tablesorter.parsers.js (hunk 1 above)
Comment (tulinkry) [inline web/js/tablesorter.parsers.js]: I've added them

**Comment by tulinkry** [comment]
Issue #1238

**Pull request merged**